### ISSUE TITLE
Exclude period

### ISSUE THREAD

**Issue by atiberghien** [django-recurrence/django-recurrence#116]
Hi,

I would like to modelize a simple use cas: a shop opened from 01/01/2018 to 31/12/2018 **except** during the period going from the 01/07/2018 to the 31/12/2018.

Several others issues are relating to the fact that there is no `dtstart` in a `recurrence.Rule `. As documented in code, it's not defined in the RFC even if it's possible with `dateutil.rrule`.

So, how can I modelize my use-case : an excluded period nested into a another ?

Cheers

**Comment by cleder** [comment]
I do not think it should go into 'django_recurrence' if it is not RFC compliant. This will cause problems with other products like `django_ical`

For your specific use case i would try to store the exclusion period in another field and pass it to the 'dateutil.rrule` before calculating the recurrences.
